Zyantific.Zydis/Native/Zydis.cs

```diff
@@ -1,7 +1,34 @@
-﻿namespace Zyantific.Zydis.Native
+﻿using System.Runtime.InteropServices;
+
+using ZyanStatus = System.UInt32;
+using ZyanU64 = System.UInt64;
+
+namespace Zyantific.Zydis.Native
 {
-    public class Zydis
+    public enum ZydisFeature
     {
+        ZYDIS_FEATURE_AVX512,
+        ZYDIS_FEATURE_KNC,
+        ZYDIS_FEATURE_MAX_VALUE = ZYDIS_FEATURE_KNC,
+    }
+
+    [StructLayout(LayoutKind.Sequential)]
+#if ZYDIS_UNSAFE
```

Comment:
The unsafe block is not neccessary, if the class does not contain any unsafe code.